README.md

```diff
@@ -41,22 +41,24 @@ If you're testing a Phoenix application with Ecto then you can enable concurrent
 ```elixir
 # lib/endpoint.ex
 
-if Application.get_env(:your_app, :sql_sandbox) do
-  plug Phoenix.Ecto.SQL.Sandbox
-end
+defmodule YourApp.Endpoint do
+  use Phoenix.Endpoint, otp_app: :your_app
+
+  if Application.get_env(:your_app, :sql_sandbox) do
+    plug Phoenix.Ecto.SQL.Sandbox
+  end
 ```
 
+Make sure Phoenix is set up to serve endpoints:
+
 ```elixir
 # config/test.exs
 
-# Make sure Phoenix is setup to serve endpoints
 config :your_app, YourApplication.Endpoint,
   server: true
-
-config :your_app, :sql_sandbox, true
```

Comment:
I've added the line back, but still extracted the instructions from the example to the main text, and added some words indicating that the SQL sandbox needs to be enabled.